ddl/options_test.go

```diff
@@ -15,22 +15,21 @@
 package ddl_test
 
 import (
+	"github.com/stretchr/testify/require"
```

Comment:
```
[2021-11-30T08:07:27.397Z] ddl/options_test.go:18: File is not `goimports`-ed (goimports)
[2021-11-30T08:07:27.397Z] 	"github.com/stretchr/testify/require"
```

let's not do the test-infra migration in this PR.

Reply:
ok